smalltalksrc/VMMakerTests/VMPrimitiveTest.class.st

```diff
@@ -4830,7 +4837,7 @@ VMPrimitiveTest >> testSumSmallIntegersWithoutOverflow [
 	ifOverflowValue := memory integerObjectOf: 99.
 	expectedResult := memory integerObjectOf: 52.
 	
-	result := interpreter sumSmallInteger: a withSmallInteger: b ifOverflow: [ ifOverflowValue ].
+	result := interpreter sumSmallInteger: a withSmallInteger: b ifOverflow: [ ^ ifOverflowValue ].
```

Comment:
Yes, this (the old version of this test) is the test that breaks with the new version of the code :)